### PR TITLE
Fix/error msg for unsuccessful fluent launch

### PR DESCRIPTION
**Pull request by prmukherj**
The following error was handled to give a proper-error msg:
error_to_be_fixed

### CODE REVIEW

File: tests/test_session.py [hunk 1]
```diff
@@ -202,3 +203,11 @@ def test_get_fluent_mode(new_mesh_session):
     assert session.fluent_connection.get_current_fluent_mode() == "meshing"
     session = session.switch_to_solver()
     assert session.fluent_connection.get_current_fluent_mode() == "solver"
+
+
+def test_unsuccessful_fluent_connection():
+    # start-timeout is intentionally provided to be 2s for the connection to fail
+    logging.disable(logging.CRITICAL)
```
Comment (mkundu1) [inline tests/test_session.py:210]: why this is needed?

File: src/ansys/fluent/core/session.py [hunk 2]
```diff
@@ -25,11 +25,14 @@
...
-    port = int(ip_and_port[1])
-    password = lines[1].strip()
-    return ip, port, password
+    try:
+        ip_and_port = lines[0].strip().split(":")
+        ip = ip_and_port[0]
+        port = int(ip_and_port[1])
+        password = lines[1].strip()
+        return ip, port, password
+    except Exception:
```
Comment (seanpearsonuk) [inline src/ansys/fluent/core/session.py:34]: @prmukherj the correct catch-all type is `BaseException`.

File: src/ansys/fluent/core/session.py [hunk 3]
```diff
@@ -25,11 +25,14 @@
 def parse_server_info_file(filename: str):
```
Comment (seanpearsonuk) [inline src/ansys/fluent/core/session.py:25]: @prmukherj when this function is called the connection has already timed out. Currently, we simply log an error, break out of the connection loop and call this function. Instead we should simply raise an exception at the point of the error (failed connection). We don't need to change this code.

**Review by seanpearsonuk — CHANGES_REQUESTED** [review]
Please try the changes mentioned in my comment.

File: tests/test_session.py (hunk 1 above)
Comment (prmukherj) [reply tests/test_session.py:210]: @mkundu1, it was logging and error of time-out.

File: src/ansys/fluent/core/session.py (hunk 2 above)
Comment (mkundu1) [reply src/ansys/fluent/core/session.py:34]: Note that, catching `BaseExecption` will shallow Ctrl+C. Catching `Exception` is generally fine for all coding logic level exception.

File: tests/test_session.py (hunk 1 above)
Comment (prmukherj) [reply tests/test_session.py:210]: this was given just so that line is not logged during testing.

File: src/ansys/fluent/core/session.py (hunk 3 above)
Comment (prmukherj) [reply src/ansys/fluent/core/session.py]: @seanpearsonuk, Thank you for the suggestion. This makes more sense. Changes applied.

File: tests/test_session.py (hunk 1 above)
Comment (mkundu1) [reply tests/test_session.py:210]: Ok, probably a good idea to enable it after the test

Comment (prmukherj) [reply tests/test_session.py:210]: @mkundu1, ohk, thank you Mainak. This has been applied.

Comment (seanpearsonuk) [reply tests/test_session.py:210]: @prmukherj with the new fix in place, we no longer need this.

File: src/ansys/fluent/core/launcher/launcher.py [hunk 4]
```diff
@@ -446,7 +446,9 @@ def launch_fluent(
                     break
                 if start_timeout == 0:
                     LOG.error("The launch process has been timed out.")
```
Comment (seanpearsonuk) [inline src/ansys/fluent/core/launcher/launcher.py:448]: @prmukherj do we need to log the error as well as raise?

File: tests/test_session.py (hunk 1 above)
Comment (prmukherj) [reply tests/test_session.py:211]: This test has been skipped altogether now. We can run this only locally.

File: src/ansys/fluent/core/launcher/launcher.py (hunk 4 above)
Comment (prmukherj) [reply src/ansys/fluent/core/launcher/launcher.py]: @seanpearsonuk, the Runtime error msg has been updated with the error log msg and the log has been deleted.

**Review by seanpearsonuk — APPROVED** [review]

**Review by mkundu1 — APPROVED** [review]

**Pull request merged**